locations/spiders/shopee_xpress_tw.py

```diff
@@ -0,0 +1,56 @@
+import json
+from typing import Any
+
+from scrapy import Spider
+from scrapy.http import Response
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+SHOPEE_XPRESS = {"brand_wikidata": "Q109676747"}
+SIMPLE_MART = {"brand_wikidata": "Q15914017"}
+
+
+class ShopeeXpressTWSpider(Spider):
+    name = "shopee_xpress_tw"
```

Comment:
If https://github.com/osmlab/name-suggestion-index/pull/10854 gets merged this this should be renamed.